theme/templates/base.html

```diff
@@ -196,7 +196,7 @@
                         <h3>{{ siteconf.col1_heading }}</h3>
                     </div>
                     <div class="content">
-                        {{ siteconf.col1_content|richtext_filters|safe }}
+                        {{ siteconf.col1_content|richtext_filters }}
```

Comment:
I wonder if we can re-use the comment filter here since that likely allows some formatting html tags